bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/pageobjects/touchui/ConfigDialog.java

```diff
@@ -99,6 +100,21 @@ public void configureWith(ComponentConfiguration config) {
     confirm();
   }
 
+  public DialogField getFieldOnTab(String label, String tab, Class fieldType) {
```

Comment:
This method is imho redundant: in case of filling in multiple fields you will be doing the following:
```
dialog.switchTab("tab1");
dialog.getField("label", "type").set();
dialog.getField("label", "type").set();
dialog.getField("label", "type").set();
dialog.getField("label", "type").set();
```
rather than:
```
dialog.getFieldOnTab("label", "tab1", "type1").set();
dialog.getFieldOnTab("label", "tab1", "type2").set();
dialog.getFieldOnTab("label", "tab1", "type3").set();
dialog.getFieldOnTab("label", "tab1", "type4").set();
```